docs/guide/troubleshooting.asciidoc

```diff
@@ -0,0 +1,25 @@
+[[troubleshooting]]
+== Troubleshooting
+
+If you run into trouble, there are three places you can look for help.
+
+[float]
+=== Troubleshooting documentation
+
+The APM Server and some of the APM agents have troubleshooting guides:
+
+* {apm-server-ref-v}/troubleshooting.html[Server troubleshooting]
+* {apm-java-ref-v}/trouble-shooting.html[Java agent troubleshooting]
+* {apm-node-ref-v}/troubleshooting.html[Node.js agent troubleshooting]
+* {apm-rum-ref-v}/troubleshooting.html[RUM troubleshooting]
+
+[float]
+=== GitHub repositories
+
+Elastic APM is open source! Our https://github.com/elastic?utf8=%E2%9C%93&q=apm[GitHub repositories] have a plethora of knowledge available in the form of code, issues, and PRs.
```

Comment:
GH repos are typically for "verified bug reports, feature requests, and pull requests." - can we break this out from the troubleshooting section?